packages/next/client/image.tsx

```diff
@@ -251,6 +252,7 @@ export default function Image({
       paddingBottom: `${ratio}%`,
     }
     imgStyle = {
+      visibility: 'hidden',
```

Comment:
```suggestion
      visibility: lazy ? 'hidden' : 'visible',
```

I think you need to test `loading=eager` to make sure that works too